src/project-tests/pomodoro-clock-tests.js

```diff
@@ -511,6 +516,37 @@ export default function createPomodoroClockTests() {
         });
       });
 
+      it(`27. The audio element with id="beep" must be 1 second or longer.`,
+      function() {
+        FCC_Global.assert.isAbove(
+          document.getElementById('beep').duration,
```

Comment:
What are the chances that .duration will return NaN? This could need to wait for the https://developer.mozilla.org/en-US/docs/Web/Events/canplaythrough event listener, but I'm not sure. The tests pass.